templates/nro/.circleci/config-header.yml.tmpl

```diff
@@ -5,8 +5,8 @@ version: 2.1
 parameters:
   release_stage:
     type: enum
-    default: "staging"
-    enum: ["staging", "production", "rollback"]
+    default: "release"
+    enum: ["release", "rollback"]
```

Comment:
Minor semantic comment: Now that we don't have two different release stages we could rename that parameter to a less confusing name. eg. `pipeline_name`, `deploy_name`, `deploy_stage`, etc

Reply:
I agree, but maybe better to take this up separately afterwards? Changing this needs to be matched with changes in backstop so it's a bit more complex to apply.